torch/fx/graph_module.py

```diff
@@ -568,7 +568,8 @@ def _gen_model_repr(module_name: str, module: torch.nn.Module) -> Optional[str]:
                 torch.save(module, module_file)
                 blobified_modules.append(module_name)
                 module_repr = module.__repr__().replace("\r", " ").replace("\n", " ")
-                module_str = f"torch.load(r'{module_file}') # {module_repr}"
+                # weights_only=False as this is legacy code that saves the model
+                module_str = f"torch.load(r'{module_file}', weights_only=False) # {module_repr}"
```

Comment:
@Chillee Wondering whether there is any BC concern here (tagging you as I noticed you added this API in https://github.com/pytorch/pytorch/pull/47544).

For context --  we are planning to flip the default for `torch.load` from `weights_only=False` --> `weights_only=True` in a future release. This PR is checking what will fail in existing CI and it seems this call needs `weights_only=False` as it serializes loads the serialized module.

Do you think there are existing modules dumped with `to_folder` that might break? 

I notice that above this says `@is_backward_compatible(False)`, so are we ok with breaking these?

Reply:
From offline discussion -- this is ok